src/otel-collector/otelcol-elastic-otlp-config.yaml

```diff
@@ -136,281 +121,13 @@ processors:
           enabled: true
         os.type:
           enabled: true
-  transform:
-    error_mode: ignore
-    trace_statements:
-      - context: span
-        statements:
-          # could be removed when https://github.com/vercel/next.js/pull/64852 is fixed upstream
-          - replace_pattern(name, "\\?.*", "")
-          - replace_match(name, "GET /api/products/*", "GET /api/products/{productId}")
-  # [Elastic Trace Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/elastictraceprocessor)
-  elastictrace: {} # The processor enriches traces with elastic specific requirements.
-  # [LSM Interval Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/lsmintervalprocessor)
-  lsminterval:
```

Comment:
Just to make sure I understand it correctly: these are not needed, because this sends to an EDOT hosted on elastic cloud (with serverless or classic) and that collector has these components, so we can drop them here - this collector just forwards OTLP. Is that the case?

Reply:
Yes :)